example/js/app-webpack/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "my-webpack-app",
+  "version": "1.0.0",
+  "description": "A web application using example ICU4X bindings, using webpack.",
+  "type": "module",
+  "scripts": {
+    "clean": "rm dist/*.js dist/*.wasm",
+    "build": "npx webpack"
```

Comment:
Suggestion: try `npm run webpack` or `./node_modules/bin/webpack`. `npx` is good for one-off commands on the command line, but I think it's not what you want in a build script, because you should instead download your dependencies in devDependencies and then have this all be local.